src/lib/FieldType/Mapper/ImageFormMapper.php

```diff
@@ -68,6 +89,21 @@ public function configureOptions(OptionsResolver $resolver)
                 'translation_domain' => 'ibexa_content_type',
             ]);
     }
+
+    /**
+     * @return array<string>
+     */
+    private function getMimeTypesChoiceList(): array
+    {
+        $mimeTypeChoiceList = [];
+        foreach ($this->allowedMimeTypes as $mimeType) {
+            $extensions = implode(', ', $this->mimeTypes->getExtensions($mimeType));
```

Comment:
In my opinion we should display extensions in the following format `*.png, *.jpg` to indicate the list contains actual file extensions. Also it would help to avoid displaying labels like `png (png)`, instead you'll get `png (*.png)`.

Reply:
I like this, but I will ask Ida for confirmation before I provide changes.

Reply:
Approved from Design Team
https://github.com/ibexa/admin-ui/pull/1021/commits/697a859eb62c069e473d7a51e89225c7e519c395